client/src/pages/Expenses/Expenses.tsx

```diff
@@ -54,13 +38,16 @@ const Expenses = () => {
     >
       <AnimatePresence>
         {displayAdder && (
-          <ExpenseAdder setDisplayAdder={setDisplayAdder} addItem={addItem} />
+          <ExpenseAdder
+            setDisplayAdder={setDisplayAdder}
+            toggleChange={toggleChange}
+          />
         )}
       </AnimatePresence>
       <ExpenseTracker
         setDisplayAdder={setDisplayAdder}
-        deleteItem={deleteItem}
         filteredExpenseData={filteredExpenseData}
+        toggleChange={toggleChange}
```

Comment:
Same here, toggle change is not a very specific name.  It is not very readable.

Reply:
okay - would something like const [rerender, setRerender] be better?

I was just stumped on how I can actually trigger a re-render once we add/delete items.